### DISCUSSION TITLE
Itwin Exporter for Datasmith (Navisworks 2024)

### DISCUSSION THREAD

**Discussion by Dom24123** [iTwin/community#8] [Q&A]
Itwin Exporter for Datasmith
The exporter is not taking the timeline sequence from Navisworks 2024. I want the sequence for Construction Animation in order to input into Unreal Engine. The schedule does not show in the itwin model view or in Unreal?

**Comment by MichaelBelousov** [comment]
Hi @Dom24123,

We launched the iTwin Exporter for Datasmith last year as a Technology Preview; we received great feedback from early adopters. One of the most significant drawbacks of the iTwin Exporter for Datasmith that we received feedback about was that the file format is unsuitable for large-scale infrastructure projects. One of the core tenets of the iTwin Platform is scalable support for large-scale modern infrastructure projects. With this limitation identified, we have developed the 3DFT technology, which supports large-scale infrastructure model streaming in Unreal Engine. 3DFT is replacing the iTwin Exporter for Datasmith, and this is a bug in the iTwin Exporter for Datasmith.

To get started with 3DFT, register an application on the iTwin Platform by following [these steps](https://developer.bentley.com/tutorials/quickstart-web-and-service-apps/#12-register-your-application).

You can get more information about how to load your iTwin in Unreal at our [Unreal plugin's page](https://github.com/iTwin/unreal-engine-3dft-plugin)

Regarding your initial question, note that the iTwin connector for NavisWorks does not support navisworks timeline sequences.

**Comment by Dom24123** [comment]
Hi Michael
Thank you for the response on the matter this clarifies my understanding of the software greatly.

if I go the route of using 3DFT this will this enable the Navisworks timeliner schedule animation of construction to enter into Unreal Engine, or is this only possible with SYNCHRO?

**Reply by MichaelBelousov** [reply]
The short answer is no, not currently.

AFAIK, iTwin's navisworks connector doesn't support timelines, but I will ask someone who knows more than I to chime in.

3DFT support for schedule animation is also a work in progress, so that is not yet available even if you had SYNCHRO data.

**Reply by MichaelBelousov** [reply]
@timlawrence-bentley do you know if the navisworks connector supports importing timelines into the iTwin format?
@Dom24123 you can request that feature here: https://itwin.ideas.aha.io/ideas

**Comment by timlawrence-bentley** [comment]
Hi,

The Navisworks connector does not support timelines yet. We are waiting for support for this to be added into a 3rd party library we use to read the Navisworks models. We have a very rough estimate that it will be available in the connector at the end of the year, but that is subject to that support being added into that library.

I’ll keep you posted.

Thanks,

Tim

**Comment by Dom24123** [comment]
Thank you gentleman for your detailed feedback, this is much appreciated. I am currently looking into 3DFT.

As i have invested a lot of time and effort in this i really want to see it come to fruition, Is there a possible trial version of Synchro Pro 4D that can demonstrate this capability therefore providing the encouragement and backing for the company to obtain the license.
